packages/wallet-connect-sdk-core/src/index.ts

```diff
@@ -122,7 +357,22 @@ export class WcSdk {
         return await WcSdk.invokeFunction(this.wcClient, this.session, this.chainId, request)
     }
 
-    async testInvoke(request: ContractInvocation) {
+    /**
+     * Sends a `testInvoke` request to the Wallet and it will communicate with the blockchain.
+     * It will not consume any gas but it will also not persist any data, this is often used to retrieve SmartContract information or check how much gas an invocation will cost.

```

Comment:
```suggestion
     * It will not consume any gas but it will also not persist any data. This is often used to retrieve SmartContract information or check how much gas an invocation will cost.

```